components/ui/pathogen-map/pathogen-map.tsx

```diff
@@ -65,10 +87,32 @@ export function PathogenMap<
     return;
   }
 
-  const layerForCountryHighlighting = layers.find((layer): layer is PathogenMapLayerInfoWithCountryHighlighting<TPathogenDataPointProperties> =>
-    shouldLayerBeUsedForCountryHighlighting(layer)
+  const layerForCountryHighlighting = layers.find(
+    (
+      layer
+    ): layer is PathogenMapLayerInfo =>
+      shouldLayerBeUsedForCountryHighlighting(layer)
   );
```

Comment:
This variable's only use is being passed as `dataLayer` to the country highlight layer. We're no longer using that prop in the country highlight layer because we're assuming `dataPoints` correlates to what we want to use for the country highlighting so this variable is unused now.

Reply:
so for this, I can see that is being used for something called a positionedUnderLayerWithId. Not sure what it is but we can remove it in the future if needed. Will leave a comment